bridge-fungible-token/src/bridge_fungible_token.sw

```diff
@@ -91,13 +91,13 @@ impl BridgeFungibleToken for Contract {
     #[storage(read, write)]
     fn claim_refund(originator: b256, asset: b256) {
         let stored_amount = storage.refund_amounts.get((originator, asset));
```

Comment:
Should we just unwrap at this line?

Reply:
Yep, makes sense as we then need to use the unwrapped value more than once after this.